<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncuestadorPickForm</class>
 <widget class="QDialog" name="EncuestadorPickForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>158</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>298</width>
    <height>158</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nuevo Encuestador</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="tituloLbl">
     <property name="text">
      <string>Titulo:</string>
     </property>
     <property name="buddy">
      <cstring>tituloCmb</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="tituloCmb"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nombreLbl">
     <property name="text">
      <string>Nombres:</string>
     </property>
     <property name="buddy">
      <cstring>nombreEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nombreEdit">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="apellidoLbl">
     <property name="text">
      <string>Apellidos:</string>
     </property>
     <property name="buddy">
      <cstring>apellidoEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="apellidoEdit">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="agregarButton">
       <property name="text">
        <string>Agregar</string>
       </property>
       <property name="icon">
        <iconset resource="../centrohistorico.qrc">
         <normaloff>:/icons/agregar.png</normaloff>:/icons/agregar.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarButton">
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="../centrohistorico.qrc">
         <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../centrohistorico.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>agregarButton</sender>
   <signal>clicked()</signal>
   <receiver>EncuestadorPickForm</receiver>
   <slot>submit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelarButton</sender>
   <signal>clicked()</signal>
   <receiver>EncuestadorPickForm</receiver>
   <slot>revert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>submit()</slot>
  <slot>revert()</slot>
 </slots>
</ui>
